src/main/java/io/getunleash/FakeUnleash.java

```diff
@@ -107,17 +107,6 @@ public void disableAllExcept(String... excludedFeatures) {
         }
     }
 
-    @Override
-    public Variant deprecatedGetVariant(String toggleName, UnleashContext context) {
```

Comment:
This is a public interface change. This was in place to support legacy evaluation of variants. We're removing that in the next major, as per https://github.com/Unleash/unleash-client-java/issues/234